### PR TITLE
No more secret hard coded

### PR DESCRIPTION
**Pull request by Azecko**
"secret" hard coded is bad for going in production.
Now, a strong password is generated using the `secrets` library.

Also added a quick debug for the PHP command so it's easier to debug if we got errors.

### CODE REVIEW

File: wpn-kopf.py [hunk 1]
```diff
@@ -126,13 +127,14 @@ def install_wordpress_via_php(self, path, title, tagline, plugins, unit_id, lang
...
                                f"--db-user={self.prefix['user']}{self.name}",
-                               f"--db-password=secret",
+                               f"--db-password={secret}",
                                f"--title={title}",
                                f"--tagline={tagline}",
                                f"--plugins={plugins}",
                                f"--unit-id={unit_id}",
                                f"--languages={languages}",
                                f"--secret-dir={Config.secret_dir}"], capture_output=True, text=True)
+      logging.debug(f"ensure-wordpress-and-theme.php --name={self.name} --path={path} --wp-dir={Config.wp_dir} --wp-host={Config.wp_host} --db-host={Config.db_host} --db-name={self.prefix['db']}{self.name} --db-user={self.prefix['user']}{self.name} --db-password=xxx --title={title} --tagline={tagline} --plugins={plugins} --unit-id={unit_id} --languages={languages} --secret-dir={Config.secret_dir}")
```
Comment (domq) [inline wpn-kopf.py:137]: Please create a wrapper of `subprocess.run()` that takes a `log=True` optional argument, rather than duplicating the command line. https://en.wikipedia.org/wiki/Don%27t_repeat_yourself (Also, this belongs in another PR)

**Review by domq — CHANGES_REQUESTED** [review]

File: wpn-kopf.py (hunk 1 above)
Comment (Azecko) [reply wpn-kopf.py]: It's a great remark. I removed the commit and I will do what you told in a second time in another PR. Thanks !
